website/src/css/custom.css

```diff
@@ -2143,4 +2143,66 @@ table th {
 [data-theme='dark'] table th {
   color: #000000; /* Black text on darker background */
   font-weight: bold;
-}
\ No newline at end of file
+}
+
+.tabs {
+  background-color: transparent;
+  padding: 10px;
+}
+
+/* Default color and border for tabs */
+.tabs__item {
+  background-color: #f9fbfc; /* Sets the tab color the same as the file borders */
```

Comment:
In an effort to reduce the amount of custom colors used, would we be able to swap this with the `var(--color-white)` variable. That color is close enough to white where it shouldn't be too noticeable.